libs/partners/google-genai/langchain_google_genai/chat_models.py

```diff
@@ -139,7 +140,7 @@ def _chat_with_retry(**kwargs: Any) -> Any:
     return _chat_with_retry(**kwargs)
 
 
-async def _achat_with_retry(*, generation_method: Callable, **kwargs: Any) -> Any:
+async def _achat_with_retry(generation_method: Callable, **kwargs: Any) -> Any:
```

Comment:
Kinda nice to keep the * to require keyword args, but since it's a private method it's not that big a deal